app/res/values/idp_configs.xml

```diff
@@ -1,19 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!--
-    This contains the authorization service configuration details that are used to demonstrate
-    authentication. By default, all authorization services are disabled until you modify this file
-    to provide your own configuration details.
-    -->
-    <eat-comment/>
+    <bool name="openid_enabled">true</bool>
 
-    <bool name="google_enabled">false</bool>
-    <string name="google_client_id" translatable="false">YOUR_ID.apps.googleusercontent.com</string>
-    <!--
-    NOTE: This scheme is automatically provisioned by Google for Android OAuth2 clients, and is
-    the reverse form of the client ID registered above. Handling of this scheme is registered in an
-    intent filter in the app's manifest.
-    -->
-    <string name="google_auth_redirect_uri" translatable="false">com.googleusercontent.apps.YOUR_ID:/oauth2redirect</string>
+    <string name="openid_client_id" translatable="false">client_id</string>
+    <string name="openid_client_secret" translatable="false">client_secret</string>
 
-</resources>
+    <string name="openid_auth_redirect_uri" translatable="false">appscheme://client.example.com</string>
```

Comment:
The placeholder redirect should `com.example.app:/provider`.  

One `/` after the hose, since there's no authority, and the custom scheme needs to be in reverse DNS notation to ensure it's IANA compliant for provide-use schemes relevant docs: https://tools.ietf.org/html/draft-ietf-oauth-native-apps-07#section-7.1 & https://tools.ietf.org/html/draft-ietf-oauth-native-apps-07#section-9